### ISSUE TITLE
Count matrix with and without lammano workflow

### ISSUE THREAD

**Issue by njbernstein** [pachterlab/kallisto#318]
Hi there,

If I run the same data through `kb` with and without `--workflow lammano` and of course the necessary associated files should I get back the same exact count matrix. For the count matrix here I am talking about `X` in the .`h5ad` object generated. 

Best,
Nick

**Comment by Yenaled** [comment]
No, you will not be getting the same exact count matrix. When you do pseudoalignment with introns included, your numbers will be different.

**Comment by njbernstein** [comment]
@Yenaled  why is that?

**Comment by Yenaled** [comment]
Your index is different. Some reads that couldn't map anywhere with a cDNA-only index will now map somewhere when you include introns. Some reads that mapped to a transcript with a cDNA-only index may now no longer map anywhere (because some k-mers in the read that originally didn't map may now map to an intronic region of some other transcript, causing the intersection of the k-compatibility classes associated with the read to be zero). Thus, your read counts will be different (not drastically so, but they will be different).
